tools/acorn-optimizer.mjs

```diff
@@ -2029,6 +2029,7 @@ try {
     preserveParens: closureFriendly,
     onComment: closureFriendly ? sourceComments : undefined,
     sourceType: exportES6 ? 'module' : 'script',
+    allowAwaitOutsideFunction: exportES6,
```

Comment:
I think perhaps we should just unconditionally allow this.  We don't rely on acorn to enforce these kind of rules and we don't know where the final code will be run.

Reply:
I agree, yeah, that seems simpler and better,
```suggestion
    allowAwaitOutsideFunction: true,
```

Reply:
Incorporated with  commit 8191423281151961b4fe8b521206a96214a2db9a, thanks!